### PR TITLE
OSOE-158: Typo fixes

### PR DESCRIPTION
**Pull request by Luko6**
OSOE-158

### CODE REVIEW

File: Lombiq.Tests.UI/Services/AccessibilityCheckingConfiguration.cs [hunk 1]
```diff
@@ -45,10 +45,10 @@ public class AccessibilityCheckingConfiguration
     /// Gets or sets a predicate that determines whether accessibility checking and asserting the results should run for
     /// the current page. This is only used if <see cref="RunAccessibilityCheckingAssertionOnAllPageChanges"/> is set to
     /// <see langword="true"/>. Defaults to <see
-    /// cref="EnableOnValidatablePagesAccessbilityCheckingAndAssertionOnPageChangeRule"/>.
+    /// cref="EnableOnValidatablePagesAccessibilityCheckingAndAssertionOnPageChangeRule"/>.
     /// </summary>
-    public Predicate<UITestContext> AccessbilityCheckingAndAssertionOnPageChangeRule { get; set; } =
-        EnableOnValidatablePagesAccessbilityCheckingAndAssertionOnPageChangeRule;
+    public Predicate<UITestContext> AccessibilityCheckingAndAssertionOnPageChangeRule { get; set; } =
```
Comment (Piedone) [inline Lombiq.Tests.UI/Services/AccessibilityCheckingConfiguration.cs:58]: This is a breaking change, so re-add the property with the old name with `[Obsolete]`, and let it just proxy to this one.

**Review by Piedone — CHANGES_REQUESTED** [review]

File: Lombiq.Tests.UI/Services/AccessibilityCheckingConfiguration.cs (hunk 1 above)
Comment (Luko6) [reply Lombiq.Tests.UI/Services/AccessibilityCheckingConfiguration.cs:58]: I couldn't find a way without making the correctly named method static, which is unsafe so, I added this back the way it was with obsolete.

Comment (Piedone) [reply Lombiq.Tests.UI/Services/AccessibilityCheckingConfiguration.cs:58]: But that won't work, since nothing uses it.

Comment (Luko6) [reply Lombiq.Tests.UI/Services/AccessibilityCheckingConfiguration.cs:58]: What do you recommend then?

Comment (0liver) [reply Lombiq.Tests.UI/Services/AccessibilityCheckingConfiguration.cs:58]: Intrigued by your comments, I checked and this works for me:
```c#
    public Predicate<UITestContext> AccessbilityCheckingAndAssertionOnPageChangeRule
    {
        get => AccessibilityCheckingAndAssertionOnPageChangeRule;
        set => AccessibilityCheckingAndAssertionOnPageChangeRule = value;
    }

    public Predicate<UITestContext> AccessibilityCheckingAndAssertionOnPageChangeRule { get; set; } =
        EnableOnValidatablePagesAccessbilityCheckingAndAssertionOnPageChangeRule;
```

Comment (Piedone) [reply Lombiq.Tests.UI/Services/AccessibilityCheckingConfiguration.cs:58]: This, i.e. a proxy property, is what I've recommended.

**Review by Piedone — APPROVED** [review]

File: Lombiq.Tests.UI/Services/AccessibilityCheckingConfiguration.cs [hunk 2]
```diff
@@ -41,6 +41,7 @@ public class AccessibilityCheckingConfiguration
     /// </summary>
     public bool RunAccessibilityCheckingAssertionOnAllPageChanges { get; set; }
 
+    // Once this method is removed, remove 'Accessbility' from the spelling whitelist.
```
Comment (Piedone) [inline Lombiq.Tests.UI/Services/AccessibilityCheckingConfiguration.cs]:
```suggestion
    // Once this method is removed, remove 'Accessbility' from the spelling whitelist in GitHub Actions.
```

**Review by Piedone — CHANGES_REQUESTED** [review]

**Review by Piedone — APPROVED** [review]

**Pull request merged**